test/tests/parallel/set3/scripts/helper_functions.bats

```diff
@@ -1,23 +1,63 @@
-load "${REPOSITORY_ROOT}/test/helper/setup"
 load "${REPOSITORY_ROOT}/test/helper/common"
 
-BATS_TEST_NAME_PREFIX='[Scripts] (helper functions inside container) '
-CONTAINER_NAME='dms-test_helper_functions'
+BATS_TEST_NAME_PREFIX='[Scripts] (helper functions) '
+SOURCE_BASE_PATH="${REPOSITORY_ROOT:?Expected REPOSITORY_ROOT to be set}/target/scripts/helpers"
 
-function setup_file() {
-  _init_with_defaults
-  _common_container_setup
+@test '(network.sh) _sanitize_ipv4_to_subnet_cidr' {
+  source "${SOURCE_BASE_PATH}/network.sh"
+
+  run _sanitize_ipv4_to_subnet_cidr '255.255.255.255/0'
+  assert_output '0.0.0.0/0'
+
+  run _sanitize_ipv4_to_subnet_cidr '192.168.255.14/20'
+  assert_output '192.168.240.0/20'
+
+  run _sanitize_ipv4_to_subnet_cidr '192.168.255.14/32'
+  assert_output '192.168.255.14/32'
+}
+
+@test '(utils.sh) _env_var_expect_zero_or_one' {
+  source "${SOURCE_BASE_PATH}/log.sh"
+  source "${SOURCE_BASE_PATH}/utils.sh"
+
+  export ZERO=0
+  export ONE=1
+  export TWO=2
+
+  run _env_var_expect_zero_or_one ZERO
+  assert_success
+
+  run _env_var_expect_zero_or_one ONE
+  assert_success
+
+  run _env_var_expect_zero_or_one TWO
+  assert_failure
+  assert_output --partial "The value of 'TWO' is not zero or one ('2'), but was expected to be"
+
+  run _env_var_expect_zero_or_one
+  assert_failure
+  assert_output --partial "ENV var name must be provided to _env_var_expect_zero_or_one"
 }
 
-function teardown_file() { _default_teardown ; }
+@test '(utils.sh) _env_var_expect_integer' {
+  source "${SOURCE_BASE_PATH}/log.sh"
+  source "${SOURCE_BASE_PATH}/utils.sh"
+
+  export INTEGER=1234
+  export NEGATIVE=-${INTEGER}
+  export NaN=not_an_integer
+
+  run _env_var_expect_integer INTEGER
+  assert_success
 
-@test "_sanitize_ipv4_to_subnet_cidr" {
-  _run_in_container_bash "source /usr/local/bin/helpers/index.sh; _sanitize_ipv4_to_subnet_cidr 255.255.255.255/0"
-  assert_output "0.0.0.0/0"
+  run _env_var_expect_integer NEGATIVE
+  assert_success
 
-  _run_in_container_bash "source /usr/local/bin/helpers/index.sh; _sanitize_ipv4_to_subnet_cidr 192.168.255.14/20"
-  assert_output "192.168.240.0/20"
+  run _env_var_expect_integer NaN
```

Comment:
Same here (variables / export needed?).

Reply:
Export is actually not required, you're right.

Reply:
Resolved in [dae7229](https://github.com/docker-mailserver/docker-mailserver/pull/3261/commits/dae722954c61a6c64084961d80e95c844559b50f)